api/v1/storagecluster_types.go

```diff
@@ -137,11 +140,41 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ExternalStorageKind specifies a kind of the external storage
+type ExternalStorageKind string
+
+const (
+	// KindOcs specifies a "ocs" kind of the external storage
+	KindOcs ExternalStorageKind = "ocs"
+
+	// KindRhcs specifies a "rhcs" kind of the external storage
+	KindRhcs ExternalStorageKind = "rhcs"
+)
+
 // ExternalStorageClusterSpec defines the spec of the external Storage Cluster
-// to be connected to the local cluster
+// to be connected to the external cluster
 type ExternalStorageClusterSpec struct {
-	// +optional
+	//+kubebuilder:default:=true
+	// Enable enables the external mode
 	Enable bool `json:"enable,omitempty"`
+
+	//+kubebuilder:default:=rhcs
+	// StorageProviderKind describes the kind of the external storage
+	StorageProviderKind ExternalStorageKind `json:"storageProviderKind,omitempty"`
+
+	// ConnectionString describes the blob of provider
+	ConnectionString string `json:"connectionString,omitempty"`
+
+	//+kubebuilder:default:="1Ti"
```

Comment:
@nb-ohad I am thinking to remove this `kubebuilder:default` tag as it does not make sense in case of the rhcs and can mislead.

Reply:
So how are we going to handle an empty value? are we going to hard code the 1TB value in the operator itself?

Reply:
No, create a CR with the value specified.

Reply:
@iamniting See my response below to @nbalacha comment around default values

Reply:
> So how are we going to handle an empty value? are we going to hard code the 1TB value in the operator itself?

I was thinking of the other way in the reconcile, CR validation as it is done in ocs-operator in many cases.

Reply:
@iamniting Please refer to the discussion below. 
I think as it stands now we need a 3rd opinion on the matter. Already pinged Umanaga and/or Jose